### ISSUE TITLE
Long term support: jemalloc for epel8

### ISSUE THREAD

**Issue by ingvarha** [jemalloc/jemalloc#1608]
Hello, everybody

EPEL-8 was recently released. EPEL is "Extra Packages for Enterprise Linux", a Fedora project to build and maintain stable packages for fixed versions of Red Hat Enterprise Linux and clones like CentOS over the whole lifecycle of that OS. This means picking a version of a package needs some consideration. EPEL strives for continous API and ABI stability for perhaps 10 years or more. It is for example only OK to upgrade a dependency package if the result is just a  restart of depending software. Exceptions arise from time to time of course, for example with non-patchable vulnerabilities, but the main rule is a stable environment.

epel5, 6, and 7 had jemallox-3.6. It seems it is time to upgrade.

jemalloc-5.2.y, or perhaps even 5.x.y looks promising. Are there any plans of long term support for the jemalloc 5.x.y series?

Ingvar

**Comment by ingvarha** [comment]
@jasone Do you have any other comments on this?

EPEL #bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1756993

**Comment by jasone** [comment]
I agree with using 5.2.x as the basis for the next long-term support version.  It captures the stabilization of 5.x prior to major new feature work.
